great-idea-website/css/index.css

```diff
@@ -63,4 +63,131 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+
+.container {
+	width: 880px;
+	/* TB LR */
```

Comment:
Great use of a container, starting off your project already set to cascade down and not have to worry about it later. good job.